cmd/minikube/cmd/start.go

```diff
@@ -173,6 +173,15 @@ func runStart(cmd *cobra.Command, args []string) {
 		os.Setenv(k, v)
 	}
 
+	//Create the host directories if not present
+	hostDirs := []string{viper.GetString(hostConfigDir), viper.GetString(hostDataDir), viper.GetString(hostVolumesDir)}
+	err = cluster.EnsureHostDirectories(libMachineClient, hostDirs)
+	if err != nil {
+		// TODO glog is probably not right here. Need some sort of logging wrapper
```

Comment:
Did you cut&paste this comment? I think we can skip this. We know that we have to revisit this.